sql/2021/http/response_headers_largest.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+SELECT
+  IF(ENDS_WITH(_TABLE_SUFFIX, 'desktop'), 'desktop', 'mobile') AS client,
+  SUBSTRING(url, 1, 400) AS url,
+  respHeadersSize / 1024 AS respHeadersSizeKiB
+FROM
+  `httparchive.summary_requests.*`
+WHERE
+  respHeadersSize IS NOT NULL AND
+  _TABLE_SUFFIX LIKE "2021_07_01%"
```

Comment:
Any reason not to do it this way?
```suggestion
  `httparchive.summary_requests.2021_07_01_*`
WHERE
  respHeadersSize IS NOT NULL
```

Reply:
None whatsoever! Think I copied from the historical one (`response_headers_size_growth.sql`) and just limited to this one. Your way is better so will update (though think BigQuery is clever enough to effectively do the same so no query difference).